templates/cronjob.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cleanupdisk
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  schedule: "0 10 * * *"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: cleanupdisk
+        spec:
+          serviceAccountName: {{ template "fullname" . }}
+          restartPolicy: Never
+          containers:
+          - image: softonic/kube-gcp-disks-roomba:{{ .Values.image.tag }}
+            args:
+	    {{- range .Values.args }}
```

Comment:
Why is this red?

Reply:
It had no default value.
Fixed in the following commit.

Reply:
This continues red